nntrainer/tensor/bcq_tensor.cpp

```diff
@@ -399,9 +401,15 @@ void BCQTensor::createBCQW() {
   /// found with various values according to the usage environment.
   size_t hidden_tile_size = 32;
 
+  // bcq_weight = std::make_shared<BiQGEMM::BCQW>(
+  //   (uint32_t *)getData(), (float *)getScale(), width(), height(),
+  //   number_of_cluster, qbit_of_clusters, size_of_clusters, hidden_tile_size);
```

Comment:
```suggestion
  // bcq_weight = std::make_shared<BiQGEMM::BCQW>(
  //   (uint32_t *)getData(), (float *)getScale(), width(), height(),
  //   number_of_cluster, qbit_of_clusters, size_of_clusters, hidden_tile_size);
```

let's remove this line

Reply:
Okay. i will remove